lib/data/sql/V1.1__import_lu.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE lu_standorttypen (STO_Nr TEXT, STO_DEU TEXT, STO_LAT TEXT, EIGNUNG TEXT, WB_VERJ_ENT TEXT, WB_PFL TEXT, BESCHREIBUNG TEXT, HOEHENVERBREITUNG TEXT, VEGETATION TEXT, VORWALDBAUMARTEN TEXT, GESGR_Nr TEXT, VERDRISK TEXT, PRIORITAET TEXT);
+COPY lu_standorttypen
+FROM '/data/profiles/lu/lu_standorttypen.csv'
```

Comment:
```suggestion
FROM '/data/profiles/lu/standorttypen.csv'
```

The prefix is a bit redundant.

Reply:
Removed